service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpConnectionHandler.java

```diff
@@ -121,7 +121,6 @@ public void onConnectionBound(Event event)
 
             SslDomain domain = makeDomain();
             domain.setPeerAuthentication(SslDomain.VerifyMode.VERIFY_PEER);
-            Ssl ssl = transport.ssl(domain);
```

Comment:
keep the "transport.ssl(domain);" part. Just because we don't use the return type doesn't mean it isn't doing anything

Reply:
Great catch.